kythe/go/platform/tools/kzip/mergecmd/flag.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mergecmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"kythe.io/kythe/go/util/vnameutil"
+)
+
+// vnameRules is a path-valued flag used for loading VName mapping rules from a vnames.json file.
+type vnameRules struct {
+	filename string
+	vnameutil.Rules
+}
+
+// Set implements part of the flag.Value interface and will append a new value to the flag.
```

Comment:
It's possible but not very common in Go to have repeated flags accumulate this way. Do we need more than one at all, and can we do it with (say) comma-separated filenames instead?

Relatedly: It's a little weird to keep only the last filename. Do we need the filename at all?

Reply:
This comment was mistaken (and copied from `flag.go` in `createcmd`).  It just sets a new filename to parse.  The value of the flag is the filename, we just read it as part of parsing the flags.

As to the other comment, accumulating flags *is* used by the `create` because it otherwise very quickly becomes incredibly unwieldy to actually use. ISTR a whole review thread on that.

Reply:
👍 Re: Accumulating flags. It's not necessarily always a bad idea, just something to be deliberate about. I don't intend to re-litigate the other flag. 😀

I would probably have realized it was a copy issue if I'd actually read the code more carefully 👀.

Reply:
No worries, I'm just glad you're still paying attention :-)